packages/contracts/contracts/FeeManager.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract FeeManager is Ownable {
+    uint256 public version = 1;
```

Comment:
How does this versioning work? Is it used for anything?

Reply:
Not yet, a stable versioning set-up is at the top of the current to-do list before this is merged